risc0/zkvm/methods/rand/Cargo.lock

```diff
@@ -354,6 +354,12 @@ dependencies = [
  "either",
 ]
 
+[[package]]
+name = "lazy_static"
+version = "1.4.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "e2abad23fbc42b3700f2f279844dc832adb2b2eb069b2df918f455c4e18cc646"
```

Comment:
I wonder why this is showing up in all the guest lock files?

Reply:
I added `lazy_static`as a dependency of `risc0_groth16`, which is a dependency of `risc0_zkvm` (used to verify the Groth16 receipts).

Reply:
Is it possible to use `OnceCell` now that this is in stable rust? I think I recently went thru and replaced all lazy static with that.